modules/ceph/src/test/java/org/testcontainers/containers/CephContainerTest.java

```diff
@@ -0,0 +1,98 @@
+package org.testcontainers.containers;
+
+import com.amazonaws.ClientConfiguration;
+import com.amazonaws.Protocol;
+import com.amazonaws.services.s3.AmazonS3;
+import com.amazonaws.services.s3.AmazonS3ClientBuilder;
+import com.amazonaws.services.s3.model.Bucket;
+import com.amazonaws.services.s3.model.S3Object;
+import org.apache.commons.io.IOUtils;
+import org.junit.Before;
+import org.junit.Rule;
+import org.junit.Test;
+
+import java.io.IOException;
+import java.io.InputStream;
+import java.net.URL;
+import java.nio.charset.StandardCharsets;
+import java.time.Duration;
+import java.time.Instant;
+import java.util.Date;
+import java.util.List;
+
+import static org.junit.Assert.*;
+
+public class CephContainerTest {
+
+    @Rule
+    public CephContainer cephContainer = new CephContainer()
+            .withAwsAccessKey("test")
+            .withAwsSecretKey("test")
+            .withBucketName("test")
+            .withStartupTimeout(Duration.ofMinutes(5));
+
+    private AmazonS3 amazonS3;
+
+    @Before
+    public void setUp() {
+        amazonS3 = AmazonS3ClientBuilder.standard()
+                .withCredentials(cephContainer.getAWSCredentialsProvider())
+                .withEndpointConfiguration(cephContainer.getAWSEndpointConfiguration())
+                .withPathStyleAccessEnabled(true)
+                .withClientConfiguration(
+                        new ClientConfiguration()
+                                .withProtocol(Protocol.HTTP)
+                                .withSignerOverride("S3SignerType")
+                )
+                .build();
+    }
+
+    @Test
+    public void testS3Bucket() {
```

Comment:
I think this test does too much, so the Ceph integration we only need to check that API works in general :)

Reply:
I added these tests for an example of connecting s3-sdk to ceph.